src/main/java/net/sf/rails/game/GameManager.java

```diff
@@ -115,9 +115,7 @@
      */
     protected GuiHints guiHints;
 
-    /**
-     * Flags to be passed to the UI, aiding the layout definition
-     */
+    /** Flags to be passed to the UI, aiding the layout definition */
```

Comment:
We should decide on a code style here. I normally use multi-line comments for class definitions, class fields and class methods. Only inside methods I use single-line comments.

What about you @jklap @neutronc?

Reply:
I tend to use multiple for class and methods, single line for inside methods.

For class fields I tend to also stick with single line unless it's long enough where it makes the comment too long. Multiple line for class fields just seems to take up a lot of space and doesn't really add value unless class and method where you should have parameters, return details, class description etc.

Reply:
agreed

Reply:
When do you use:

```
// ...
```

compared to:

```
/** ... */
```

?

Reply:
Class and methods only. Even multiline comments inside of methods I use `//` as `/* */` is just plain harder to scan/read especially on a single line. The benefit of `/* */` is when you have that multiline comment as you don't need to preface every line with a `//` BUT in those cases using the opening `/*` and the closing `*/` tend to be on their OWN lines to increase the readability of the comment. In a method this just adds extra lines that get in the way (vs the top of a method or class where it is not that important because you aren't trying to follow code flow.